include/quda.h

```diff
@@ -87,6 +87,8 @@ extern "C" {
     int return_result_gauge; /**< Return the result gauge field */
     int return_result_mom;   /**< Return the result momentum field */
 
+    int use_fgi;             /**< add FGI terms */
```

Comment:
Perhaps this should be a `bool` instead?  (Note that I'm going to kill `QudaBoolean` at some point, and replace it with `bool` - this is fine since `bool` is supported on C99 which we require anyway for the QUDA C interface.)